docs/diagrams/AddState1.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+'https://plantuml.com/class-diagram
+
+hide circle
+hide methods
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+skinparam ClassBackgroundColor #FFFFAA
+
+title initial state
+
+class uniquePersonList as "<u>uniquePersonList:UniquePersonList</u>"
+class student1 as "<u>student1:Person</u>" {
+    name = Alice
+    id = A0234567X
+    email = e0234567@u.nus.edu
+    grade = A
+    group = Group 1
```

Comment:
Should these be shown as separate objects?

Reply:
> Should these be shown as separate objects?

Thanks for notifying. I have created a pr to update the AddState diagram.